radix-transaction-scenarios/src/runners/dumper.rs

```diff
@@ -200,7 +200,7 @@ mod test {
     #[test]
     pub fn check_state_and_event_hashes_for_up_to_anemone_scenarios_on_bottlenose() {
         assert_event_and_state_hashes(
-            "9d06891ba213a57e14031af89cd21b040a1e0c7b362f5e425d1dd281e258d47d",
+            "3f80c8cdcd6785cb2339792f567258bb1b1ccf00932505b606e2ae1385c4fbb3",
```

Comment:
The two hashes for `up_to_anemone` and `up_to_bottlenose` should not change.

Reply:
This is still bottlenose protocol though no?

Reply:
Josh is right. These are run `on_bottlenose`. I think these tests could be quite a lot cleaner.

I was hoping to try to neaten them up over the next couple of weeks if I have some time; to make their intent clearer.